source/administration/command-line-tools.rst

```diff
@@ -1495,6 +1496,23 @@ mattermost team list
     .. code-block:: none
 
       ./mattermost team list
+      
+mattermost team modify
+~~~~~~~~~~~~~~~~~~~~~~
+
+  Description
+    Modify a team's privacy setting to public or private.  
```

Comment:
Propose description consistent with the comparable command for `mattermost channel modify` https://docs.mattermost.com/administration/command-line-tools.html#mattermost-channel-modify

Reply:
@jasonblais thanks and updated.